src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +10,12 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private final UserService userService = new UserService();
+
     public boolean login(String email, String password) {
+        if (userService.findByEmail(email) != null) {
```

Comment:
How can you avoid calling findByEmail method twice here?

Reply:
i try to avoid it by saving user in login method. i make new pull request